packages/bsky/src/db/tables/actor.ts

```diff
@@ -2,7 +2,7 @@ export interface Actor {
   did: string
   handle: string | null
   indexedAt: string
-  takedownId: number | null // @TODO(bsky)
+  takedownId: string | null // @TODO(bsky)
```

Comment:
This TODO was way back from the appview split, I don't think it's relevant anymore.
```suggestion
  takedownId: string | null
```